lib/kafo/exit_handler.rb

```diff
@@ -1,11 +1,10 @@
 module Kafo
   class ExitHandler
-    attr_accessor :cleanup_paths, :exit_code, :logger
```

Comment:
this drops `logger=`, which could still be easily added by `attr_writer :logger`, but if we don't use it anywhere (e.g. tests) I have no problem with it

Reply:
I think the logger should not be part of the public API. Tests create a new exit handler every time so it shouldn't be that relevant.

Reply:
Ack